### PR TITLE
add goal from my time page

### LINKED ISSUE #1958: Add a Goal from My Time page
**Is your feature request related to a problem? Please describe.**
It's very annoying when you open the app to quickly add a Goal, and first have to switch to My Time page to add a Goal.

**Describe the solution you'd like**
- Add a '+' button on the My Time page, just like on My Goals.
- When adding a Goal via My Time page, don't leave the My Time page - just show the add Goal modal
- When saving, regardless of where you are in the Goal hierarchy on the My Goals page, since you added via My Time in a hurry, there is no reasonable place to put it, other than the root level.  
This way you can add quickly, even on My Time, and sort out where you want it in you hierarchy later.

**Describe alternatives you've considered**
None.

**Additional context**
#1957 Needs to be done first.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #1958

### CODE REVIEW

**Review by tijlleenders — CHANGES_REQUESTED** [review]
Yeah this is good.

Do you also see the modal going 'off screen' at the top? It would be nice if we can force the modal to stick to the virtual keyboard. This is not the first time I see this so might be good to make a separate issue for that (although I think there was one at some point so it might be a regression).  

Apart from checking the issue history for that  
and  fixing it (or making a separate issue for it),  
- what I realize is that a sound confirmation is not enough here. I'd like something visual, like a toast message with "[first max 15 characters of goal title] created!" message, as part of this PR.

**Comment by tijlleenders** [comment]
Modal going offscreen:

**Comment by vinaybadgujar102** [comment]
@tijlleenders I will make a new issue for the modal offscreen problem. 
I have added the toast once new goal is added. You can review it.

**Review by tijlleenders — APPROVED** [review]
This works, thanks.

File: src/components/index.scss [hunk 1]
```diff
@@ -33,29 +33,43 @@
...
+    border-radius: 50%;
+    border: none;
+    background: var(--selection-color);
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+    width: 56px;
+    height: 56px;
+    color: white;
+    font-weight: 600;
+    font-size: 0.875em;
+    display: flex;
```
Comment (Tushar-4781) [inline src/components/index.scss:63]: you can use "place-middle" in class name. check in short.scss

File: src/components/index.scss [hunk 2]
```diff
@@ -33,29 +33,43 @@
...
+
+  .goal-btn-circle {
+    border-radius: 50%;
+    border: none;
+    background: var(--selection-color);
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+    width: 56px;
+    height: 56px;
+    color: white;
+    font-weight: 600;
```
Comment (Tushar-4781) [inline src/components/index.scss:61]: you can use fw-600 in class name

File: src/components/index.scss [hunk 3]
```diff
@@ -33,29 +33,43 @@
...
+    font-size: 0.875em;
```
Comment (Tushar-4781) [inline src/components/index.scss:62]: pick a font class present from short.scss

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

File: src/components/index.scss (hunk 3 above)
Comment (vinaybadgujar102) [reply src/components/index.scss]: removed font-size as it was not needed

**Review by Tushar-4781 — APPROVED** [review]

**Pull request merged**